debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -1517,11 +1517,30 @@ protected void addToTransaction(String transactionId, LogMinerEventRow row, Supp
             return;
         }
 
+        final LogMinerEvent event;
+        try {
+            event = eventSupplier.get();
+        }
+        catch (DmlParserException e) {
+            switch (connectorConfig.getEventProcessingFailureHandlingMode()) {
+                case FAIL:
+                    LOGGER.error("Failed to parse SQL for event '{}'", row);
+                    throw e;
+                case WARN:
+                    LOGGER.warn("Failed to parse SQL '{}'. The event '{}' is being ignored and skipped.", row.getRedoSql(), row);
```

Comment:
```suggestion
                    LOGGER.warn("Failed to parse SQL for event '{}'. This event is being ignored and skipped.", row.getRedoSql(), row);
```

Reply:
Hi @thibthibus, there are two arguments in this case, so you get the redo SQL when set to `WARN.`  These changes seem to ignore that fact.

Reply:
Oh my... Of course, sorry...

Reply:
Just ordered https://amzn.eu/d/daZpLfc. Apparently I need a little refresher (or to sleep more). ;-)